src/azure-cli/azure/cli/command_modules/rdbms/_params.py

```diff
@@ -23,7 +31,55 @@ def load_arguments(self, _):    # pylint: disable=too-many-statements
+                                        actions=[LocalContextAction.SET], scopes=['{} server'.format(command_group)]))
+        
+        server_name_getter_arg_type = CLIArgumentType(metavar='NAME', 
+                                        help="Name of the server. The name can contain only lowercase letters, numbers, and the hyphen (-) character. Minimum 3 characters and maximum 63 characters.",
+                                        local_context_attribute=LocalContextAttribute(name='server_name', 
+                                        actions=[LocalContextAction.GET], scopes=['{} server'.format(command_group)]))
+
+        server_name_arg_type = CLIArgumentType(metavar='NAME', 
+                                        help="Name of the server. The name can contain only lowercase letters, numbers, and the hyphen (-) character. Minimum 3 characters and maximum 63 characters.",
+                                        local_context_attribute=LocalContextAttribute(name='server_name', 
+                                        actions=[LocalContextAction.SET, LocalContextAction.GET], scopes=['{} server'.format(command_group)]))
+        
+        overriding_none_arg_type = CLIArgumentType(local_context_attribute=LocalContextAttribute(name='context_name', actions=[LocalContextAction.GET]))
+
+        with self.argument_context(command_group) as c:
+            c.argument('name', options_list=['--sku-name'], required=True)
+            c.argument('server_name', arg_type=server_name_arg_type, completer=server_completers[command_group], options_list=['--server-name', '-s'], help='Name of the server. The name can contain only lowercase letters, numbers, and the hyphen (-) character. Minimum 3 characters and maximum 63 characters.')
+
+        with self.argument_context('{} server'.format(command_group)) as c:
+            c.ignore('family', 'capacity', 'tier')
+
+            c.argument('server_name', arg_type=server_name_arg_type, options_list=['--name', '-n'], id_part='name', help='Name of the server. The name can contain only lowercase letters, numbers, and the hyphen (-) character. Minimum 3 characters and maximum 63 characters.')
+            c.argument('administrator_login', options_list=['--admin-user', '-u'], help='Administrator username for the server. Once set, it cannot be changed.')
+            c.argument('administrator_login_password', options_list=['--admin-password', '-p'], help='The password of the administrator. Minimum 8 characters and maximum 128 characters. Password must contain characters from three of the following categories: English uppercase letters, English lowercase letters, numbers, and non-alphanumeric characters.')
+            c.argument('ssl_enforcement', arg_type=get_enum_type(['Enabled', 'Disabled']), options_list=['--ssl-enforcement'], help='Enable or disable ssl enforcement for connections to server. Default is Enabled.')
```

Comment:
Actually we usually define the parameter with `action='store_true'` for this case. Considering there are so many in rdbms which defined with `arg_type=get_enum_type(['Enabled', 'Disabled'])`, I'm OK with current defintion.

Reply:
Got it. Thanks